types/react-datepicker/index.d.ts

```diff
@@ -11,131 +11,140 @@
 //                 Roman Nuritdinov <https://github.com/Ky6uk>
 //                 Avi Klaiman <https://github.com/aviklai>
 //                 Naoki Sekiguchi <https://github.com/seckie>
+//                 tu4mo <https://github.com/tu4mo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import * as React from "react";
-import * as Popper from "popper.js";
+import * as React from 'react';
+import * as Popper from 'popper.js';
```

Comment:
👋 @tu4mo could you please revert some changes caused by your local formatter? I mean `'` → `"` and extra whitespace on some lines. This should make the diff smaller and thus the changes will be easier to review.

Reply:
Hey @kachkaev! I used a separate commit for Prettier changes and you can also hide white-space changes from the cog-button. I can of course remove the formatting, but I was thinking that it would be better to conform to the Prettier config of DefinitelyTyped. What do you think?

https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.prettierrc.json

Reply:
Oh, seems like I missed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35672